tests/staggered_invert_test.cpp

```diff
@@ -607,8 +607,8 @@ invert_test(void)
   if (dslash_type == QUDA_ASQTAD_DSLASH) {
     gaugeParam.type = QUDA_ASQTAD_LONG_LINKS;
     gaugeParam.ga_pad = link_pad;
-    gaugeParam.reconstruct= link_recon;
-    gaugeParam.reconstruct_sloppy = link_recon_sloppy;
+    gaugeParam.reconstruct= (link_recon == QUDA_RECONSTRUCT_12 || link_recon == QUDA_RECONSTRUCT_8) ? QUDA_RECONSTRUCT_13 : link_recon;
```

Comment:
I don't get why we need this. Isn't specifying `--recon 13` on the command line good enough?

Reply:
I made that choice based on precedence---in `staggered_dslash_(c)test`, you specify (or, for `ctest`, it loops over) reconstruct 18, 12, or 8, and it has a similar conditional when uploading the long links.

Precedence aside, there are other places `link_recon` is used in the code where recon 13 can't be used. For example, it's used to define the reconstruct used on the original links for building the fat/long links, and in that case recon 13 gives you an error that reads something along the lines of "link recon 13 is only supported for staggered long links."

In that regard, it's an unavoidable choice, which is why I went with precedence.

Reply:
Let's just go with that then.